crates/web5/src/dids/methods/did_web/resolver.rs

```diff
@@ -34,7 +34,14 @@ impl Resolver {
         };
 
         Self {
-            did_url: format!("https://{}/did.json", did_url),
+            did_url: format!(
+                "{}://{}/did.json",
+                match did_url.contains("localhost") {
```

Comment:
should this be == localhost?
What if I have neal-localhost.com ?

Reply:
yeah we need to use the `Url` crate, but will do subsequently